CHANGES.md

```diff
@@ -30,7 +30,8 @@ next
 2.6
 ---
 * Require GHC 8.8.
-* `Sing` has switched from a data family to a type family. This has a number of
+* `Sing` has switched from a data family to a type family.  This [github issue comment](https://github.com/goldfirere/singletons/issues/318#issuecomment-467067257) provides a detailed explanation for the motivation behind this change. 
+This has a number of
```

Comment:
Add two more spaces before this line to ensure that it is properly aligned with the previous bullet point.